kalitectl.py

```diff
@@ -219,7 +225,6 @@ def get_pid():
         raise NotRunning(7)  # Unclean shutdown
 
     # TODO: why is the port in django settings!? :) /benjaoming
-    from django.conf import settings
```

Comment:
Traceback (most recent call last):
  File "kalitectl.py", line 507, in <module>
    args=arguments['DJANGO_OPTIONS']
  File "kalitectl.py", line 369, in start
    manage('kaserve', args)
  File "kalitectl.py", line 300, in manage
    utility.execute()
  File "python-packages/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "python-packages/django/core/management/__init__.py", line 272, in fetch_command
    klass = load_command_class(app_name, subcommand)
  File "python-packages/django/core/management/__init__.py", line 77, in load_command_class
    module = import_module('%s.management.commands.%s' % (app_name, name))
  File "python-packages/django/utils/importlib.py", line 36, in import_module
    __import__(name)
  File "/data/data/com.android.kalite27/files/kalite/distributed/management/commands/kaserve.py", line 18, in <module>
File "/data/data/com.android.kalite27/files/kalite/caching/__init__.py", line 27, in <module>
    from kalite import i18n, topic_tools
  File "/data/data/com.android.kalite27/files/kalite/i18n/__init__.py", line 42, in <module>
    DUBBED_VIDEOS_MAPPING_FILEPATH = os.path.join(settings.I18N_DATA_PATH, "dubbed_video_mappings.json")
  File "python-packages/django/conf/__init__.py", line 54, in __getattr__
    return getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'I18N_DATA_PATH'